guides/v2.3/cloud/project/project-conf-files_services-elastic.md

```diff
@@ -14,13 +14,13 @@ functional_areas:
 *   Supports stop words and synonyms
 *   Indexing does not impact customers until the reindex operation completes
 
-{{site.data.var.ee}} supports Elasticsearch version 5.2.
+{{site.data.var.ee}} supports Elasticsearch version 5.2 and 6.x (requires v2.3.1 or later).
 
 {:.bs-callout .bs-callout-info}
 Elasticsearch 5.2 is only available for 2.2.3 and later. If you are upgrading to {{site.data.var.ee}} 2.1.3, you must change your configuration as discussed in [the 2.1.3 Release Notes]({{ site.baseurl }}/guides/v2.1/cloud/release-notes/CloudReleaseNotes2.1.3.html#cloud-rn-213-es). See [Elasticsearch information]({{ page.baseurl }}/config-guide/elasticsearch/es-overview.html) to learn more.
 
 {:.bs-callout .bs-callout-warning}
-If you prefer using an existing search service, such as Elasticsearch, instead of relying on the default Cloud configuration, you can use the [`SEARCH_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#search_configuration) environment variable to connect it to your site.
+If you prefer to use an existing search service instead of relying on the default Cloud configuration, you can use the [`SEARCH_CONFIGURATION`]({{ page.baseurl }}/cloud/env/variables-deploy.html#search_configuration) environment variable to connect it to your site.
```

Comment:
This note is not completely accurate. There is no true warning here. The default will change in the future; this function only helps to change configuration and not the service itself. You can remove this note completely.